js_modules/dagit/packages/core/src/app/PythonErrorInfo.tsx

```diff
@@ -86,6 +97,39 @@ export const UNAUTHORIZED_ERROR_FRAGMENT = gql`
   }
 `;
 
+export const CopyErrorButton = ({copy}: {copy: () => void | string}) => {
+  return (
+    <div style={{position: 'relative'}}>
+      <CopyErrorButtonWrapper
+        onClick={async () => {
+          const message = copy();
+          await showSharedToaster({
+            message: message ?? <div>Copied value</div>,
+            intent: 'success',
+          });
+        }}
+      >
+        <Icon name="content_copy" /> Copy
+      </CopyErrorButtonWrapper>
```

Comment:
It might look fine just to use a `<Button outlined>` here.

Reply:
I think we intentionally don't want an outline here. Josh had outlined in his first design but then removed the outline